### PR TITLE
KB: Clarify the impacted version of filesystem trim

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
Add a form to clarify the impacted version

### CODE REVIEW

File: kb/2024-01-30/the_potential_risk_with_fstrim.md [hunk 1]
```diff
@@ -12,9 +12,21 @@ hide_table_of_contents: false
...
+Please check the following form for the impacted version.
+
 
+| Harvester Version | Longhorn Version | Impacted by Filesystem Trim |
+|-------------------|------------------|-----------------------------|
+| v1.2.0            | v1.4.3           | Yes                         | 
+| v1.2.1            | v1.4.3           | Yes                         |
+| v1.2.2            | v1.5.5           | No                          |
+| v1.3.0            | v1.6.0           | Yes                         |
+| v1.3.1 (upcoming) | v1.6.2           | No                          |
```
Comment (bk201) [inline kb/2024-01-30/the_potential_risk_with_fstrim.md:26]: I'd suggest we list affected versions only or the opposite way to show the fixed version.
@jillian-maroket do you have any suggestions?

**Review by jillian-maroket — COMMENTED** [review]
@Vicente-Cheng @bk201 Please check my suggestions.

**Comment by Vicente-Cheng** [comment]
Thanks for the suggestion!

Updated!

**Review by bk201 — APPROVED** [review]
LGTM, thanks.

**Review by jillian-maroket — APPROVED** [review]
LGTM. Thanks!

**Comment by Vicente-Cheng** [comment]
Just add `Co-authored-by:` with the latest update.

**Pull request merged**